mike_senkovich.js

```diff
@@ -0,0 +1,6 @@
+const a = 12;
+const b = 6;
+
+const addNum = (x, y) = {
```

Comment:
In line 4 there needs to be an arrow to signify that it is a function.
const addNum = (x, y) => {